File: uhrwerk-metastore/src/main/java/io/qimia/uhrwerk/common/metastore/config/ConfigService.java
package io.qimia.uhrwerk.common.metastore.config;

import io.qimia.uhrwerk.common.metastore.model.ConnectionModel;
import io.qimia.uhrwerk.common.metastore.model.TableModel;

public interface ConfigService {
  ConnectionModel save(ConnectionModel Connection);

  TableModel tableConfig(String area, String vertical, String name, String version);

  TableModel tableConfig(TableModel table);
}

File: uhrwerk-metastore/src/main/kotlin/io/qimia/uhrwerk/common/metastore/config/ConfigService.kt
package io.qimia.uhrwerk.common.metastore.config

import io.qimia.uhrwerk.common.metastore.model.ConnectionModel
import io.qimia.uhrwerk.common.metastore.model.TableModel

interface ConfigService {
    fun save(Connection: ConnectionModel?): ConnectionModel?
    fun tableConfig(area: String?, vertical: String?, name: String?, version: String?): TableModel?
    fun tableConfig(table: TableModel?): TableModel?
}